### ISSUE TITLE
Add support for absolute path imports for JavaScript files using jsconfig.json baseUrl

### ISSUE THREAD

**Issue by ivanfuzuli** [wallabyjs/quokka#498]
### Issue description or question
I am tring to use quokka with absolute path imports. But it isn't respects jsconfig. Is there any work around?

### Sample code

```
{
  "babel": {
    "presets": ["react-app"]
  },
  "plugins": ["jsdom-quokka-plugin"]
}
```
### Quokka.js Console Output

```
Quokka 'messageReducer.js' (node: v12.16.1, babel: v7.10.2, plugins: jsdom-quokka-plugin)​​​​
 
Cannot find module 'config/@axios' 
Require stack: 
- ./src/store/actions/messageActions.js 
- ./src/store/reducers/messageReducer.js 
```

### Code editor version

Visual Studio Code v1.46.0

### OS name and version
OSX

**Comment by smcenlly** [comment]
Right now, Quokka does not support the use of absolute path imports using `jsconfig.json`. We'll investigate adding this as a feature this week and expect to provide an updated version of Quokka that supports this in the next day or so.

**Comment by ivanfuzuli** [comment]
@smcenlly Thank you.

**Comment by smcenlly** [comment]
We have added support for the `baseUrl` and `path` settings for projects/quokka files that have an associated `jsconfig.json` configuration file. This is available in the latest version of Quokka for all editors. For VS Code the latest Quokka version is v1.0.298.

As per [Microsoft's description of jsconfig.json](https://code.visualstudio.com/docs/languages/jsconfig):

> `jsconfig.json` is a descendant of `tsconfig.json`, which is a configuration file for TypeScript. `jsconfig.json` is `tsconfig.json` with `"allowJs"` attribute set to `true`.

Internally, Quokka uses Microsoft's TypeScript library to read and validate the file and uses the same logic as Microsoft to resolve your modules based on the `baseUrl` and `paths` compilerOptions. This is important to mention as it appears from the issue description that you are using react. At this point in time, react does not fully support the `jsconfig.json` `paths` setting, while Quokka now does. You may therefore notice some discrepancies between how Quokka works and how your imports work in React. Quokka's support should be a superset of what is available in react so it is unlikely to cause you problems when you are using Quokka. The create-react-app repo has [an open issue](https://github.com/facebook/create-react-app/issues/5645) that you may review if you are interested in reading about what scenarios are not supported.

**Comment by apoorvHearth** [comment]
I made a .quokka file is my project root and used the baseUrl property to load modules from the "src" directory. But quokka was throwing cannot find module error.
Here are the two versions of .quokka file which I tried :-
1. `{
  "compilerOptions": {
    "baseUrl": "src"
  }
}`

2. `{
    "baseUrl": "src"
}`


and here is my jsconfig file:-
`{
  "include": ["src/**/*"],
  "compilerOptions": {
    "jsx": "react",
    "baseUrl": "src"
  },
  "exclude": ["node_modules", "**/node_modules/*"]
}`

@smcenlly am I doing something wrong ?

**Comment by ArtemGovorov** [comment]
@apoorvHearth Try adding `paths` to your `jsconfig` file:

```diff
{
    "include": [
        "src/**/*"
    ],
    "compilerOptions": {
        "jsx": "react",
        "baseUrl": "src",
+       "paths": {}
    },
    "exclude": [
        "node_modules",
        "**/node_modules/*"
    ]
}
```

**Comment by apoorvHearth** [comment]
Okay, I found what I was doing wrong. It was a silly mistake the jsconfig I have only included files under the src folder using the "include" key. And the file I was trying to run quokka on was residing outside the src folder.

So after moving the file into the src folder everything worked perfectly without any changes.

Just for anybodies reference:-

This is how my .quokka file looks:-

`{
  "baseUrl": "src"
}
`

and here is my jsconfig file:-
`{
  "include": ["src/**/*", "Untitled-1.js"],
  "compilerOptions": {
    "jsx": "react",
    "baseUrl": "src"
  },
  "exclude": ["node_modules", "**/node_modules/*"]
}
`
